superset/assets/backendSync.json

```diff
@@ -2493,6 +2493,64 @@
       "default": "150",
       "description": "Font size for the biggest value in the list"
     },
+    "big_num_font_size": {
```

Comment:
maybe you could make this more generic like `header_font_size` to match `subheader_font_size` below. then for `big_number` you could override the label if you want, but it won't result in new controls every time we want a font size hook

Reply:
Done!